lib/internal/errors.js

```diff
@@ -114,6 +114,54 @@ function message(key, args) {
   return String(fmt.apply(null, args));
 }
 
+const primitiveTypes = [
+  'string', 'boolean', 'number', 'object', 'undefined', 'null',
+  'object', 'symbol', 'false', 'falsy', 'function', 'uint32'
+];
+function checkExpectedTypeInitial(expectedType) {
```

Comment:
If this remains in some form, it should be made into a lint rule. Otherwise it should IMO be removed.